config/install/node.type.page.yml

```diff
@@ -1,9 +1,16 @@
 langcode: en
 status: true
-dependencies: {  }
-name: 'Basic page'
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus:
+      - main
+    parent: 'main:'
+name: FAQ
```

Comment:
Let's keep 'Basic page' name. In that case, we'll be able to use this content also for other general pages like "Terms & Conditions", "About Us", etc.